vendor/brave-ios/Ads/Generated/NativeAdsClient.h

```diff
@@ -43,5 +42,6 @@ class NativeAdsClient : public ads::AdsClient {
   void SetIdleThreshold(const int threshold) override;
   uint32_t SetTimer(const uint64_t time_offset) override;
   void ShowNotification(std::unique_ptr<ads::NotificationInfo> info) override;
+  void CloseNotification(const std::string& id) override;
```

Comment:
Needs an accompanying implementation in `vendor/brave-ios/Ads/Generated/NativeAdsClient.mm` which calls the Obj-C bridge method you added in `vendor/brave-ios/Ads/Generated/NativeAdsClientBridge.h`